src/reporting/parameter.ts

```diff
@@ -80,15 +80,6 @@ export function createStackParametersQuickSight(scope: Construct, paramGroups?:
     default: 'QuickSight Owner Principal Arn',
   };
 
-  const quickSightPrincipalParam = new CfnParameter(scope, 'QuickSightPrincipalParam', {
```

Comment:
@tyyzqmf Is it safe to create/upgrade to the latest stack without this parameter?

Reply:
@tyyzqmf already completed completed the corresponding modifications。